src/compiler/binder.ts

```diff
@@ -1428,7 +1430,7 @@ namespace ts {
         function bindModuleExportsAssignment(node: BinaryExpression) {
             // 'module.exports = expr' assignment
             setCommonJsModuleIndicator(node);
-            bindExportAssignment(node);
+            declareSymbol(file.symbol.exports, file.symbol, node, SymbolFlags.Property | SymbolFlags.Export | SymbolFlags.ValueModule, SymbolFlags.None);

```

Comment:
In part because 'module.exports' is a property, but mostly because there was some code path that was crapping out without an expected flag set, of which `property` seemed appropriate (the exact code path escapes me, as I wrote this over a week ago, and there were *a lot* of code paths).

Happy to be corrected on my usage of the mystic arts of SymbolFlags however.